.gitignore

```diff
@@ -10,6 +10,7 @@
 # Ignore the default SQLite database.
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/*sqlite3*
```

Comment:
Could we change the setup to use postgresql ? you won't see any rails project with sqlite :)